src/ctap/mod.rs

```diff
@@ -1802,6 +1820,61 @@ mod test {
         assert!(make_credential_response.is_ok());
     }
 
+    #[test]
+    fn test_non_resident_process_make_credential_credential_with_cred_protect() {
+        let mut env = TestEnv::new();
+        let mut ctap_state = CtapState::new(&mut env, CtapInstant::new(0));
+
+        let test_policy = CredentialProtectionPolicy::UserVerificationOptionalWithCredentialIdList;
+        let mut make_credential_params =
+            create_make_credential_parameters_with_cred_protect_policy(test_policy);
+        make_credential_params.options.rk = false;
```

Comment:
I think `rk=False` is the default and this is similar to the test above.

Reply:
No, `rk=True` is the default, and if otherwise the original unit test can't have passed before when credProtectPolicy is not persisted.

Reply:
Ah right, I missed that in `create_make_credential_parameters_with_cred_protect_policy`. Thanks for confirming!